### ISSUE TITLE
How to create a curve object.

### ISSUE THREAD

**Issue by jawar-cn** [iden3/snarkjs#387]
I'm trying to use snarkjs in javascript .

When I new a power of tau  
```
const zkSnark = require("snarkjs");
await zkSnark.powersOfTau.newAccumulator(curve, 14, "pot14_0000.ptau")

```
I really don't know how and where to get a curve object. 

And I find a mothod in snarkjs named `getCurveFromName` which is not defined by "exports"  in `package.json`

**Comment by phated** [comment]
Curves are created when they are needed and then they are available on the global, such as `globalThis.curve_bn128`

**Comment by jawar-cn** [comment]
I use 
`console.log(globalThis)`
and console shows 
```log
<ref *1> Object [global] {
  global: [Circular *1],
  queueMicrotask: [Function: queueMicrotask],
  clearImmediate: [Function: clearImmediate],
  setImmediate: [Function: setImmediate] {
    [Symbol(nodejs.util.promisify.custom)]: [Getter]
  },
  structuredClone: [Getter/Setter],
  clearInterval: [Function: clearInterval],
  clearTimeout: [Function: clearTimeout],
  setInterval: [Function: setInterval],
  setTimeout: [Function: setTimeout] {
    [Symbol(nodejs.util.promisify.custom)]: [Getter]
  },
  atob: [Getter/Setter],
  btoa: [Getter/Setter],
  performance: [Getter/Setter],
  fetch: [AsyncFunction: fetch],
  crypto: [Getter],
  curve_bn128: null,
  curve_bls12381: null
}

```

the curve_bn128 object is null.

**Comment by OBrezhniev** [comment]
Fixed in #512
